src/client/redirect-catcher.js

```diff
@@ -0,0 +1,29 @@
+import EventEmiter from './utils/event-emitter';
+import { parseProxyUrl } from '../utils/url';
+
+export default class RedirectCatcher extends EventEmiter {
+    constructor (codeInstrumentation) {
+        super();
+
+        this.CATCHED_EVENT = 'hammerhead|event|redirect-catched';
+
+        var locationAccessorsInstrumentation = codeInstrumentation.locationAccessorsInstrumentation;
+        var propertyAccessorsInstrumentation = codeInstrumentation.propertyAccessorsInstrumentation;
+        var HASH_RE                          = /#.*$/;
```

Comment:
I suppose it should be `const HASH_RE`? And maybe place it before the class definition, as we usually do with consts